e2e/e2e_scripts_test.go

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
```

Comment:
I think you can remove the `e2e` part of the filename since it's part of the `e2e` package?